modules/mauth-authenticator-apachehttp/README.md

```diff
@@ -6,7 +6,7 @@ This is an implementation of Medidata Authentication Client Authenticator to val
 
 1. Configuration
   * MAuth uses [Typesafe Config](https://github.com/typesafehub/config).
-  Create `application.conf` on your classpath with following content.
+  Create `application.conf` on your classpath with following content. The disable_v1 flag can be set to authenticate incoming requests with Mauth protocol V2 only, default to false.
```

Comment:
if default for `disable_v1` is `false` then it should really be part of `reference.conf`

Reply:
Also, `disable_v1` is vague and using boolean type is limiting as well. Instead, I suggest to use `enum` and more verbose values, 

May be something like:
```
enum AuthType {
    V1, V1_2, V2
}
```

Reply:
Design Decision document says:
by default clients sign requests with both x-mws-authentication and mcc-authentication
…
When clients are confident that all of the servers they contact support V2, they can configure their mauth-client to only sign requests with mcc-authentication
For signer, if the disable_v1 flag is set to false, clients will send both V1 and V2 headers. If true, clients will send V2 headers only.
For authentication, clients will authenticate with only the highest protocol version present. if the "disable_v1 flag is set to true, clients will authenticate for V2 only.